### PR TITLE
126 fix charging bool refactor

### PR DESCRIPTION
**Pull request by HamzaIqbal786110**
#126 Fix charging boolean and refactor implementation

Changed the name of isCharging to isChargingEnabled to make it more clear and changed its implementation in main

### CODE REVIEW

**Comment by dyldonahue** [comment]
There is some weird stuff going on with file changes/commits here, would like to defer to @nwdepatie as to whether any of its an issue. Might need to rebase the actual changes

File: shepherd_bms/src/main.cpp
Comment (dyldonahue) [inline shepherd_bms/src/main.cpp:98]: guessing this is supposed to be logical or || not bitwise | ? or am i missing something

File: shepherd_bms/src/compute.cpp
Comment (dyldonahue) [inline shepherd_bms/src/compute.cpp:64]: being a bit picky but i think calling it isChargerConnected is more descriptive of what this really means than isChargingEnabled

**Review by dyldonahue — CHANGES_REQUESTED** [review]
A few small picky changes and just a note to possibly rebase if needed before merging

**Review by dyldonahue — APPROVED** [review]

**Review by nwdepatie — CHANGES_REQUESTED** [review]
Yeah we gotta fix those commits before we merge it in, not sure what happened its like the PR was reopened from the same branch without any changes

**Comment by nwdepatie** [comment]
Gotta rework the commits to only include the ones you meant to change, and you should rename/squash the commits to make more sense

File: shepherd_bms/src/compute.cpp
Comment (mccauleyma) [inline shepherd_bms/src/compute.cpp:36]: Is snake_case what we should be using? Don't we use camelCase?

**Review by mccauleyma — CHANGES_REQUESTED** [review]

File: shepherd_bms/src/compute.cpp
Comment (dyldonahue) [reply shepherd_bms/src/compute.cpp:36]: should rework wording on confl but nah this is correct, snake_case for variables camelCase for functions

**Comment by dyldonahue** [comment]
is charging was kind of depreciated and will probably be fully removed

**Pull request closed without merging**
